test/unit/app/users/postNewUserDetails.test.js

```diff
@@ -173,9 +173,9 @@ describe('when entering a new users details', () => {
     });
   });
 
-  it('then it should render view if email is a blacklisted email and environment is Production', async () => {
+  it('should render with error if emailValidation is undefined and the email address is a blacklisted email', async () => {
     req.body.email = 'blacklisted.domain@hotmail.com';
-    config.toggles.environmentName = 'pr';
+    process.env.emailValidation = 'true';
```

Comment:
For this test, emailValidation should be undefined I believe? 😄 Easy thing to miss!

Reply:
Fixed!